src/components/pastProjectsComponents/ProjectBlock.astro

```diff
@@ -0,0 +1,219 @@
+                </div>
+                <h1 class="project-title">{title}</h1>
+            </div>
+            <p class="project-description">{description}</p>
+            <div class="award-container">
+                {awards.map((award: any) => 
+                    (<Award icon = {award.icon} description = {award.description} />)   
+                )}
+            </div>
+        </div>
+        <div class= "img-container">
+            <img class="project-icon" src = {icon} alt = {icon}>
+        </div>
+    </div>
+    <div class ="marker"></div>
+    <div class ="year"><p>{year}</p></div>
+</div>
+
+
+<style>
+    * {
+        padding:0;
+        margin:0;
+        box-sizing: border-box;
+    }
+    .direct-arrow {
+        position:absolute; 
+        top: 1.2rem;
+        right:1.2rem;
+        width: 2.2rem;
+        height:2.2rem;
+        opacity: 0.8;
+        cursor: pointer;
+        /* This color can be a global variable */
+    }
+
+    .marker {
+        position:absolute;
+        width: 15px;
+        height: 15px;
+        border-radius: 50%;
+        top: calc(50% + 2rem);
+        transform: translateY(-50%);
+        left:-9px;
+        background: black;
+    }
+
+    .year {
+        /* background: red; */
+        font-size: 1.125rem;
+        /* font-family: Poppins; */
+        position: absolute;
+        text-align: center;
+        justify-content: center;
+        display: flex;
+        justify-content:center;
+        width: auto;
+        height: auto;
+        top: calc(50% + 2rem);
+        transform: translateY(-50%);
+        left: calc(-5.5rem);
+        font-weight: 500;
+        font-family: "Poppins";
+    }
+
+    .direct-arrow:hover {
+        opacity: 0.6;
+    }
+
+    .outer-container {
+        width: 100%;
+        padding-left: 4rem;
+        padding-top: 4rem;
+        position: relative;
+    }
+
+    .project-container {
+        width:90%;
+        min-height:10rem;
+        background:var(--color-primary-yellow);  
+        display:flex;
+        flex-direction: row;
+        flex-wrap:wrap;
+        padding: 2.8rem;
+        border: 2px solid var(--color-primary-black);
+        /* border-radius:0.6rem; */
+        border-radius: var(--border-radius-md);
+        min-width: 340px;
+        box-shadow: rgba(0, 0, 0, 0.4) 5px 5px 5.3px;
+        position:relative;
+        z-index: 30;
+    }
+
+    .info-container {
+        display:flex;
+        flex-direction: column;
+        width:65%;
+    }
+
+    .title-container {
+        display: flex;
+        flex-direction: row;
+        gap: 1rem;
+        align-items: center;
+        flex-wrap: wrap;
+    }
+
+    .title-icon-container {
+        width: auto;
+        height: 3rem;
+        display: none;
+        /* display: flex; */
+        justify-content: center;
+        align-items: center;
+    }
+
+    .title-icon {
+        height: 100%;
+        max-width: calc(330px - 6rem);  
```

Comment:
what's the significance of both of these numbers?

generally, i think you have a lot of 'magic numbers' floating around in these styles - not necessarily terrible in css, but if you are reusing a value (for padding, width, etc) a few times, extract it out into a css variable so that it's easier to understand what it's doing (and easy to update later if necessary)